src/ai-examples/README.rst

```diff
@@ -0,0 +1,4 @@
+Microsoft Azure CLI 'AI Examples' Extension
+==========================================
+
+Improve user experince by adding AI powered examples to command help content.
```

Comment:
I would suggest you may add more content to describe you stunning features.

Reply:
Sure! I added a little bit more of a description and an example usage.